Rivet/Functions/Get-RivetConfig.ps1

```diff
@@ -272,9 +272,15 @@ function Get-RivetConfig
 
     $order = Get-ConfigProperty -Name 'DatabaseOrder' -AsArray
     $pluginModules = Get-ConfigProperty -Name 'PluginModules' -AsArray
+    $hasDatabaseOrder = $false
+
+    if( $order )
+    {
+        $hasDatabaseOrder = $true
+    }
 
     [Rivet.Configuration.Configuration]$configuration = 
-        [Rivet.Configuration.Configuration]::New($Path, $Environment, $sqlServerName, $dbsRoot, $connectionTimeout, $commandTimeout, $pluginPaths, $pluginModules)
+        [Rivet.Configuration.Configuration]::New($Path, $Environment, $sqlServerName, $dbsRoot, $connectionTimeout, $commandTimeout, $pluginPaths, $pluginModules, $hasDatabaseOrder)

```

Comment:
Pass the actual `DatabaseOrder` array, not a flag. You can create a `HasDatabaseOrder` flag on the `Configuration` class that checks if the `DatabaseOrder` array is null or empty.